src/hooks/useBalance.ts

```diff
@@ -41,33 +39,12 @@ const initValues = {
   free: new BN(ZERO) as Balance
 };
 
-const useBalance = (name: string, address: string, chain: string | undefined, providedSi: boolean = false): State[] => {
+const useBalance = (api: ApiPromise, address: string, providedSi: boolean = false): State => {
```

Comment:
I do not understand why to remove the double functionality of useBalance that can be very helpful in a project like bridges. Its more convenient to receive both amounts and chose which one to use than calling it twice. Or am I missing something?

Reply:
the reason why i dissected the logic to have specific calls per chain is because this functionality was quite tight to the select account component use case when you have the input  [addrress, chain], and you get the balances for the account and derived account.  In the `Transfer` component i realized that  i needed this logic extracted so i only have one account with one balance. 
In the previous version `Account` was retrieving always two rows: Account and derived  and in transfers sometimes i just need only one.